src/assets/wise5/authoringTool/node/edit-node-title/edit-node-title.component.html

```diff
@@ -0,0 +1,12 @@
+<mat-form-field class="step-title form-field-no-hint">
+  <mat-label *ngIf="!isGroupNode" i18n>Step Title {{ nodePosition }}:</mat-label>
+  <mat-label *ngIf="isGroupNode" i18n>Lesson Title {{ nodePosition }}:</mat-label>
+  <input
+    matInput
+    [(ngModel)]="nodeJson.title"
+    [ngModelOptions]="{ debounce: 500 }"
```

Comment:
The debounce doesn't seem to work. Maybe try performing the debounce like we do in [label-authoring.component.ts](https://github.com/WISE-Community/WISE-Client/blob/92c3c16e0fdd0c5dffaf6253c1d366440be0f63a/src/assets/wise5/components/label/label-authoring/label-authoring.component.ts#L36-L40).